waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -115,3 +130,55 @@ method getAll*(db: WakuMessageStore, onData: message_store.DataProc): MessageSto
 proc close*(db: WakuMessageStore) = 
   ## Closes the database.
   db.database.close()
+
+
+proc migrate*(db: SqliteDatabase, path: string = MIGRATION_PATH, targetVersion: int64 = USER_VERSION): MessageStoreResult[bool] = 
```

Comment:
I think `migrate` is agnostic of what table(s) it migrates? If so, this function can move out of `waku_message_store` to https://github.com/status-im/nim-waku/blob/master/waku/v2/node/storage/sqlite.nim. This will signal that it's multipurpose and related to the DB and not just a single table (excited that this can be used for `Peer`-table in future as well!).

Reply:
Makes sense, done!